src/service/viewer-impl.js

```diff
@@ -245,14 +245,21 @@ export class Viewer {
         this.prerenderSize_;
     dev().fine(TAG_, '- prerenderSize:', this.prerenderSize_);
 
+    const queryParams = parseQueryString(this.win.location.search);
+    /**
+     * Whether the AMP document is embedded in a Chrome Custom Tab.
+     * @private @const {boolean}
+     */
+    this.isCctEmbedded_ = !this.isIframed_ && queryParams['amp_gsa'] === '1' &&
+        (queryParams['amp_js_v'] || '').startsWith('a');
```

Comment:
You'll have to import the `startsWith` helper from [src/string.js](https://github.com/ryjust/amphtml/blob/428fc6e8730addd928342ccaa6e462a7926897e9/src/string.js#L72-L83)

Reply:
Imported!